jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -403,6 +403,13 @@ static int getRemotePort(Request request)
         return -1;
     }
 
+    /**
+     * Get the logical name for the server, which may differ from the actual server name
+     * @param request The request to get the server name of
+     * @return The logical name the request was sent to, which may be from the authority of the
+     *         request; the configured server authority; the actual network name of the server;
+     *         otherwise null
```

Comment:
I'd prefer to keep `@return` short and simple, and add all the description and logic of what this method does to the main javadocs.